### PR TITLE
Fixes #68026: Fixed errors about golint in pkg/kubectl/cmd/attach

### PR DESCRIPTION
**Pull request by jeffliu27**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

According to this issue: #68026, I revised some errors about golint in the package below 

pkg/kubectl/cmd/attach

**Which issue(s) this PR fixes**:
Fixes #<68026>

**Special notes for your reviewer**:
After I revised above errors, I ran hack/update-bazel.sh and hack/update-gofmt.sh to make sure I fix any problems with gofmt or bazel.
Also ran unit tests: 
pkg/kubectl/cmd/attach/attach_test.go
pkg/kubectl/cmd/run/run_test.go
@dims

```release-note
Release note: None
```

### CODE REVIEW

File: pkg/kubectl/cmd/attach/attach.go [hunk 1]
```diff
@@ -61,8 +61,8 @@ const (
 	defaultPodLogsTimeout   = 20 * time.Second
 )
 
-// AttachOptions declare the arguments accepted by the Attach command
-type AttachOptions struct {
```
Comment (prksu) [inline pkg/kubectl/cmd/attach/attach.go]: please leave this naming as before.

**Review by ahmadnurus — COMMENTED** [review]
hi @jeffliu27 thanks for your contributions

renaming *Options to Options has been rejected in the past. see https://github.com/kubernetes/kubernetes/pull/70682#pullrequestreview-174591661 and it's not following kubectl [Command implementation conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-cli/kubectl-conventions.md#command-implementation-conventions)

File: pkg/kubectl/cmd/attach/attach.go (hunk 1 above)
Comment (jeffliu27) [reply pkg/kubectl/cmd/attach/attach.go]: Sounds good, I'll make a PR for other files instead

**Pull request closed without merging**